packages/cog/src/cog/cog.ts

```diff
@@ -51,15 +51,15 @@ export async function buildCogForQuadKey(
 
     const bbox = QuadKey.toBbox(quadKey);
     const { forward } = Wgs84ToGoogle;
-    const [east, north] = forward(bbox.slice(0, 2));
-    const [west, south] = forward(bbox.slice(2));
+    const [west, north] = forward(bbox.slice(0, 2));
+    const [east, south] = forward(bbox.slice(2));
 
     const [x, y, z] = QuadKey.toXYZ(quadKey);
 
     const alignmentLevels = job.source.resolution - z;
 
     const cogBuild = new GdalCogBuilder(vrtLocation, outputTiffPath, {
-        bbox: [east, south, west, north],
+        bbox: [west, south, east, north],
```

Comment:
is a bbox meant to be WSEN or ENWS?

I think we both? possibly should just pick one.

Reply:
I don't know if we have two points (lon,lat) `[-5,-6]` and `[20, 40]` then the bbox should be `[-5,-6,20,40]`

Reply:
In other words the lesser lon, the lesser lat, the greater lon, the grater lat, so `[west, south, east, north]` 

from the rfc: https://tools.ietf.org/html/rfc7946#section-5
 
   A GeoJSON object MAY have a member named "bbox" to include
   information on the coordinate range for its Geometries, Features, or
   FeatureCollections.  The value of the bbox member MUST be an array of
   length 2*n where n is the number of dimensions represented in the
   contained geometries, with all axes of the most southwesterly point
   followed by all axes of the more northeasterly point.  The axes order
   of a bbox follows the axes order of geometries.

Reply:
so GeoJSON has a standard of west,south,east,north. I believe the codebase has used east north west south in places.

so we should possilbly look to change to WSEN.